_data/other.yml

```diff
@@ -5,10 +5,7 @@ websites:
       tfa: Yes
       sms: Yes
       software: Yes
-      email: Yes
```

Comment:
Email is still supported. See [doc link](
https://helpx.adobe.com/x-productkb/global/2-step-verification-account-security.html) and screenshot below:
![Adobe 2FA Sign In Options](https://user-images.githubusercontent.com/1897053/46568141-56013180-c8fd-11e8-96b6-cce57ae07659.png)